app/assets/javascripts/osom-tables.js

```diff
@@ -117,6 +132,32 @@
     }
   }
 
+  function save_checkboxes(container) {
+    var checked_ids;
+    checked_ids = (container.data('checked') || '').split(',');
```

Comment:
Why does this have to read the current value of the `checked` data attribute. Couldn't it just set the correct `checked` value by reading the current state of all checkboxes every time? Using `push` and `splice` to modify the existing check state seems like it could be more error prone, but I might be missing something.